src/client/routes/Vaults/index.tsx

```diff
@@ -318,7 +334,17 @@ export const Vaults = () => {
                 {
                   key: 'apy',
                   header: t('components.list-card.apy'),
-                  format: ({ apyData, apyType }) => formatApy(apyData, apyType),
+                  transform: ({ apyData, apyType, apyMetadata, address }) =>
+                    apyType === 'new' || apyType === 'n/a' || !apyMetadata ? (
```

Comment:
I don't remember this part very well, but, where are the types for apyTypes? Are this strings real types or just texts used in ui? just because if we translate them later this will crash

Reply:
Found this
```
if (apyType === 'new') return 'NEW ✨';
if (apyType === 'n/a') return 'N/A';
  ```
  
  But apyType: string, we should change this in the types itself

Reply:
This is unrelated to this PR, should be fixed in a new one, resolving conversation